content/devs.json

```diff
@@ -62,5 +62,43 @@
         "url": "mailto:me@fsaycon.dev"
       }
     ]
+  },
+  {
+    "avatar": "https://i.ibb.co/QDvNvJ8/51011835-1-1.jpg",
+    "name": "Christian Villamin",
+    "title": "Software Engineer",
+    "company": "High Output Ventures",
+    "blurb": "Hi! I have a thing for optimization and solving problems. Always tinkering and excited in ever-growing tech. I also animate stuff.",
+    "skills": [
+      "TypeScript",
+      "React",
+      "GraphQL",
+      "Apollo Server/Client",
+      "Linux",
+      "Framer Motion",
+      "NextJS",
+      "GatsbyJS",
+      "MongoDB",
+      "PostgreSQL",
+      "Prisma/Nexus"
+    ],
+    "contacts": [
+      {
+        "type": "website",
+        "url": "http://codekcv.github.io/"
+      },
+      {
+        "type": "github",
+        "url": "https://github.com/codekcv"
+      },
+      {
+        "type": "linkedin",
+        "url": "https://linkedin.com/in/codekcv/"
+      },
+      {
+        "type": "email",
+        "url": "codekcv31@gmail.com"
```

Comment:
Hi, @codekcv! Can you use a [`mailto` link](https://css-tricks.com/all-about-mailto-links/) for this?

Reply:
Okie dokie @galacemiguel.